### PR TITLE
Add camel-iec60870 wrapper

### PR DESCRIPTION
**Pull request by stataru8**
Add camel-iec60870 wrapper.
Add guava dependency feature.

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -969,25 +969,19 @@
...
-<!--        <bundle dependency='true'>mvn:io.netty/netty-transport-native-unix-common/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-resolver/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-tcnative-classes/${netty-tcnative-classes-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-iec60870/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-iec60870' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version='${netty-version}'>netty</feature>
+        <bundle dependency='true'>mvn:com.google.guava/guava/${guava-version}</bundle>
+        <bundle dependency='true'>mvn:com.google.guava/failureaccess/1.0.1</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Could you please create a guava feature with those 2 dependencies as it is already used twice in the file. Please also note that the version of the second one is `1.0.2`

Comment (essobedo) [reply features/src/main/feature/camel-features.xml]: Of course, don't hesitate to replace these 2 dependencies the new feature in both places

Comment (stataru8) [reply features/src/main/feature/camel-features.xml]: mb on the `failureaccess` version, fix applied in last commit

**Review by essobedo — APPROVED** [review]
Perfect 👍 thank you

**Pull request merged**